ieeg_visual/sub-01/ses-01/ieeg/sub-01_ses-01_task-visual_run-01_ieeg.json

```diff
@@ -1,38 +1,30 @@
 {
     "TaskName": "visual",
-    "SamplingFrequency": 3051.76,
     "Manufacturer": "Tucker Davis Technologies",
-    "ManufacturersModelName": "unknown",
+    "ManufacturersModelName": "",
```

Comment:
Wouldn't it make more sense to remove this and similar keys if the value is empty?